demos/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2021 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 8000;
+
+app.use(express.static('public'));
+
+/**
+ * Redirects to SwG Basic JS.
+ * The endpoint is compatible with the classic Swgjs local demos.
+ */
+app.get('/examples/sample-pub/redirect-to/swg-basic.js', (req, res) => {
+  res.redirect('https://news.google.com/swg/js/v1/swg-basic.js');
+});
+
+app.listen(port, () => {
+  console /*OK*/
+    .log(`Swgjs Demos are available at http://localhost:${port}`);
```

Comment:
Should this line here differentiate SwG classic & SwG basic?

Reply:
Good call, updated